src/hooks.c

```diff
@@ -66,7 +66,7 @@ static s32 cpuExecuteCall_hook(Cpu* pCPU, s32 nCount, s32 nAddressN64, s32 nAddr
     s32 nDeltaAddress;
     s32 nAddressGCNCall;
 
-#if IS_OOT
+#if IS_OOT || IS_GC && IS_MM
     nCount = OSGetTick();
 #elif IS_MM
```

Comment:
nit: maybe convert these `#elif IS_MM`s to `#else` (originally I wrote `#elif IS_MM` to make it easier to read but it's no longer an oot/mm split here)